src/Terminal/HelpText.ts

```diff
@@ -281,13 +281,13 @@ export const HelpTexts: Record<string, string[]> = {
     "Usage: grow",
     " ",
     "Spoof transactions in the current server. Increasing the money available by hacking. Requires root access.",
-    "See the wiki page for hacking mechanics.",
+    "For additional information please go to Documentation > Resources > NS API Documentation > Grow for more details",
```

Comment:
```suggestion
    "For additional information please go to Documentation > Resources > NS API Documentation > Grow.",
```
"For additional information" and "for more details" are the same thing, having both is redundant. (For all three.)